web/web/package-lock.json

```diff
@@ -0,0 +1,10074 @@
+{
```

Comment:
Where did this package-lock.json come from? You should not be using `npm`. And `yarn` generates a `yarn.lock` and that should be used. Please update your internal dev flow and make sure the readme is accurate.

Reply:
removed the file